src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key in state) {
```

Comment:
Let's go through the keys that we need to remove here, instead of go through all state keys

Reply:
@Anna-Tkachenko thanks, cnanged that code